### PR TITLE
Improvement: Added separate file for info-box and icon-list attributes

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
- Added separate file for info-box and icon list block attribute and removed from block.php

### Types of changes
non-breaking change

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — APPROVED** [review]

**Review by sandeshjangam — CHANGES_REQUESTED** [review]
$defaults = UAGB_Helper::$block_list['uagb/icon-list-child']['attributes'];

This line still present

File: lib/astra-notices/class-astra-notices.php [hunk 1]
```diff
@@ -116,7 +116,7 @@ public function dismiss_notice() {
 			}
 
 			if ( false === wp_verify_nonce( $nonce, 'astra-notices' ) ) {
-				wp_send_json_error( esc_html_e( 'WordPress Nonce not validated.', 'ultimate-addons-for-gutenberg' ) );
+				wp_send_json_error( esc_html_e( 'WordPress Nonce not validated.' , 'ultimate-addons-for-gutenberg') );
```
Comment (sandeshjangam) [inline lib/astra-notices/class-astra-notices.php:119]: WHy this extra sapces

File: lib/gutenberg-templates/classes/class-ast-block-templates.php [hunk 2]
```diff
@@ -366,7 +366,7 @@ public function template_importer() {
 		public function template_assets() {
 
 			$post_types = get_post_types( array( 'public' => true ), 'names' );
-
+
```
Comment (sandeshjangam) [inline lib/gutenberg-templates/classes/class-ast-block-templates.php:369]: WHy this extra sapces

File: includes/blocks/icon-list-child/frontend.css.php [hunk 3]
```diff
@@ -7,10 +7,6 @@
  * @package uagb
  */
 
-$defaults = UAGB_Helper::$block_list['uagb/icon-list-child']['attributes'];
-
```
Comment (supriyajjadhav) [inline includes/blocks/icon-list-child/frontend.css.php:11]: @sandeshjangam Already Removed

File: lib/gutenberg-templates/classes/class-ast-block-templates.php (hunk 2 above)
Comment (sandeshjangam) [inline lib/gutenberg-templates/classes/class-ast-block-templates.php:369]:
```suggestion

```

File: lib/astra-notices/class-astra-notices.php (hunk 1 above)
Comment (sandeshjangam) [inline lib/astra-notices/class-astra-notices.php:119]:
```suggestion
				wp_send_json_error( esc_html_e( 'WordPress Nonce not validated.', 'ultimate-addons-for-gutenberg' ) );
```

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**